azure-pipelines-ci.yml

```diff
@@ -0,0 +1,56 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - '*'
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      endpoint: Planning-Inspectorate
+      name: Planning-Inspectorate/common-pipeline-templates
+      ref: refs/tags/release/3.11.0
+
+extends:
+  template: stages/wrapper_ci.yml@templates
+  parameters:
+    gitFetchDepth: 0
+    globalVariables:
+      - template: azure-pipelines-variables.yml@self
+    validationJobs:
+      - name: Run Linting & Tests
+        steps:
+          - script: git fetch origin main:main
+            displayName: Fetch main branch
+          - template: ../steps/node_script.yml
+            parameters:
+              script: npm ci
+          - template: ../steps/node_script.yml
+            parameters:
+              script: cd packages/applications-service-api && npx prisma generate
+          - template: ../steps/node_script.yml
+            parameters:
+              condition: ne(variables['Build.SourceBranchName'], 'main')
+              script: npm run commitlint
+          - template: ../steps/node_script.yml
+            parameters:
+              condition: succeededOrFailed()
+              script: npm run lint
+          - template: ../steps/node_script.yml
+            parameters:
+              condition: succeededOrFailed()
+              script: npm run test:cov
+          - template: ../steps/check_changed_files.yml
+            parameters:
+              pathFilters:
+                - name: api
+                  path: packages/applications-service-api
+                - name: common
+                  path: packages/common
+                - name: web
+                  path: packages/forms-web-app
+                - name: back-office-subscribers
+                  path: packages/back-office-subscribers
+              stepName: check_files
```

Comment:
I don't think we need this, I think it was used by the build pipeline to check what to build.